app/src/main/java/emu/skyline/input/onscreen/OnScreenControllerView.kt

```diff
@@ -258,6 +258,32 @@ class OnScreenControllerView @JvmOverloads constructor(context : Context, attrs
         invalidate()
     }
 
+    fun increaseOpacity() {
+        for (button in controls.allButtons) {
+            var currentOpactity = button.config.opacity
+            currentOpactity += 10
+            if (currentOpactity > 100) currentOpactity = 100
+            button.config.opacity = currentOpactity
+            if (button is JoystickButton) {
+                button.innerButton.config.opacity = currentOpactity
+            }
+        }
+        invalidate()
+    }
+
+    fun decreaseOpacity() {
+        for (button in controls.allButtons) {
```

Comment:
Use `Array.forEach` instead of a for-in loop:
`controls.allButtons.forEach { ... }`.